### PR TITLE
Add namespace to seperate google pubsub environment

### PR DESCRIPTION
**Pull request by jonathanhaposan**

### CODE REVIEW

File: examples/pubsub/pubsub_google.go [hunk 1]
```diff
@@ -37,7 +38,7 @@ func mainGoogle() {
 			Concurrent: 3,
 		},
 	},
-		pubsub.GoogleJSONFile("gcp-project-id", "/path/to/google/application/credentials/cred.json"))
+		pubsub.GoogleJSONFile("gcp-project-id", "environment-name", "/path/to/google/application/credentials/cred.json"))
```
Comment (budiryan) [inline examples/pubsub/pubsub_google.go]: environment / namespace ?

File: pubsub/worker_google.go [hunk 2]
```diff
@@ -33,6 +35,7 @@ func newGoogleWorker(projectID, credFile string, maxInFlight int) *googleWorker
 			googleJSONFile: credFile,
 		}),
 		maxOutstanding: maxInFlight,
+		namespace:      namespace,
```
Comment (budiryan) [inline pubsub/worker_google.go]: go fmt

File: pubsub/option.go [hunk 3]
```diff
@@ -20,6 +20,9 @@ type config struct {
 
 	// projectID specifies Google Pubsub project-id
 	projectID string
+
+	// namespace separator
```
Comment (budiryan) [inline pubsub/option.go]: add Google Pubsub to the comment

File: pubsub/worker_google.go (hunk 2 above)
Comment (jonathanhaposan) [reply pubsub/worker_google.go]: already go fmt

File: pubsub/option.go [hunk 4]
```diff
@@ -39,9 +42,10 @@ func GoogleJSON(projectID string, cred []byte) Option {
 }
 
 // GoogleJSON configures Publisher & Subscriber for Google Cloud Pub/Sub auth using JSON filename
-func GoogleJSONFile(projectID, filename string) Option {
+func GoogleJSONFile(projectID, filename, namespace string) Option {
```
Comment (febytanzil) [inline pubsub/option.go]: example & this func mismatch args

File: examples/pubsub/pubsub_google.go (hunk 1 above)
Comment (jonathanhaposan) [reply examples/pubsub/pubsub_google.go]: update to namespace

File: pubsub/option.go (hunk 3 above)
Comment (jonathanhaposan) [reply pubsub/option.go]: updated

File: pubsub/option.go (hunk 4 above)
Comment (jonathanhaposan) [reply pubsub/option.go]: updated

Comment (febytanzil) [reply pubsub/option.go]: `projectID, namespace string, cred []byte`
`projectID, filename, namespace string`
would be better with consistent args ordering

Comment (jonathanhaposan) [reply pubsub/option.go]: updated
will use `projectID, namespace string, credential`

**Pull request merged**